lang/fpc/Portfile

```diff
@@ -21,38 +21,37 @@ long_description    Free Pascal is a 32, 64 and 16 bit professional Pascal \
                     and Android.
 homepage            https://www.freepascal.org
 
-# TODO: add support for i386
-supported_archs     x86_64 arm64 ppc ppc64
+supported_archs     x86_64 arm64 i386 ppc ppc64
 
```

Comment:
That's the default value of `supported_archs` so it should be removed.
```suggestion
```

Reply:
Ah. OK. My misunderstanding the difference between commit message and pull request message. Both done with next commit.